apps/challenges/views.py

```diff
@@ -2997,6 +3000,14 @@ def manage_worker(request, challenge_pk, action):
 
     challenge = get_challenge_model(challenge_pk)
 
+    if parse(challenge["end_date"]) < datetime.utcnow() and action in ("start", "stop", "restart"):
+        response_data = {
+            "error": "The action {} is invalid for worker as the challenge ended on {}.".format(
+                action, challenge["end_date"]
```

Comment:
Change the error message to `Action {} worker is not supported for an inactive challenge`